pkg/shell/shell.scss

```diff
@@ -438,62 +429,6 @@ thead.credential-lock.credential-unlocked > *:last-child {
     cursor: auto;
 }
 
-/* server overview dashboard */
-#server .info-table-ct button {
-    max-width: 300px;
-}
-
-#server .info-table-ct td:first-child {
-    max-width: 50%
-}
-
-#server .info-table-ct {
-    width: 100%;
-}
-
-@media (min-width: 992px) {
-    #server .info-table-ct {
-        table-layout: fixed;
-        width: 100%;
-        min-width: 220px;
-    }
-
-    /* server overview dashboard */
-    #server .info-table-ct-container td button {
-        max-width: 100%;
-    }
-}
-
-@media (max-width: 991px) {
-    #server .info-table-ct-container {
```

Comment:
We could also drop it from pkg/lib/table.css then?

Reply:
Not sure what you mean. I removed '#server', not `.info-table-ct-container`. (Maybe that is also obsolete? Haven't checked)

Reply:
Yes, I don't  see any use of .info-table-ct-container anywhere.